app/controllers/application_controller.rb

```diff
@@ -5,11 +5,19 @@ class ApplicationController < ActionController::Base
   before_action :authenticate_user!
   before_action :redirect_to_setup
   protect_from_forgery with: :exception
+  # https://bugzilla.suse.com/show_bug.cgi?id=1044237
+  before_action :prevent_page_cache
 
   private
 
   def redirect_to_setup
     return true unless signed_in?
     redirect_to setup_path if Minion.assigned_role.count.zero?
   end
+
+  def prevent_page_cache
+    response.headers["Cache-Control"] = "no-cache, no-store"
+    response.headers["Pragma"] = "no-cache"
```

Comment:
This is only necessary to browsers that doesn't support HTTP 1.1. Are we supporting it? All the latest browsers are already implementing HTTP 2, I guess we can get rid of 1.0 support. Just guessing...